apps/studio/components/interfaces/Storage/EditBucketModal.tsx

```diff
@@ -16,256 +42,332 @@ import { useBucketUpdateMutation } from 'data/storage/bucket-update-mutation'
-                            >
-                              {Object.values(StorageSizeUnits).map((unit: string) => (
-                                <Listbox.Option key={unit} label={unit} value={unit}>
-                                  <div>{unit}</div>
-                                </Listbox.Option>
-                              ))}
-                            </Listbox>
-                          </div>
-                          {IS_PLATFORM && (
-                            <div className="col-span-12 mt-2">
-                              <p className="text-foreground-light text-sm">
-                                Note: Individual bucket upload will still be capped at the{' '}
-                                <Link
-                                  href={`/project/${ref}/settings/storage`}
-                                  className="font-bold underline"
-                                >
-                                  global upload limit
-                                </Link>{' '}
-                                of {formattedGlobalUploadLimit}
-                              </p>
-                            </div>
-                          )}
-                        </div>
+                          </FormControl_Shadcn_>
+                        </FormItemLayout>
                       )}
-                    </div>
-                    <Input
-                      id="allowed_mime_types"
-                      name="allowed_mime_types"
-                      layout="vertical"
-                      label="Allowed MIME types"
-                      placeholder="e.g image/jpeg, image/png, audio/mpeg, video/mp4, etc"
-                      labelOptional="Comma separated values"
-                      descriptionText="Wildcards are allowed, e.g. image/*. Leave blank to allow any MIME type."
                     />
+                    {hasFileSizeLimit && (
+                      <div className="grid grid-cols-12 col-span-12 gap-x-2 gap-y-1">
+                        <div className="col-span-8">
+                          <FormField_Shadcn_
+                            key="formatted_size_limit"
+                            name="formatted_size_limit"
+                            control={form.control}
+                            render={({ field }) => (
+                              <FormItemLayout
+                                name="formatted_size_limit"
+                                description={`Equivalent to ${convertToBytes(
+                                  formattedSizeLimit,
+                                  selectedUnit as StorageSizeUnits
+                                ).toLocaleString()} bytes.`}
+                              >
+                                <FormControl_Shadcn_>
+                                  <Input_Shadcn_
+                                    id="formatted_size_limit"
+                                    aria-label="File size limit"
```

Comment:
because we don't wrap this input with a label, we should at least give it an `aria-label` attribute describing what it is for screen readers. it also helps us to target the input with `screen.getByLabelText` in testing